config/default.json

```diff
@@ -10,6 +10,9 @@
     "useSSL": "false",
     "timeout": 1200
   },
+  "frontend": {
+    "inactivityAlert": 900
```

Comment:
will need to add this to `custom-environment-variables.json` as well.

Reply:
done 👍